frontend/src/components/UserSignUp/SignUpForm.jsx

```diff
@@ -120,43 +158,86 @@ function SignUpForm({ selectedEvent }) {
       .max(10, "Too long")
       .matches(phoneRegExp, "Invalid format"),
   });
-  const { handleSubmit, control, reset } = useForm({
+  const { handleSubmit, control, reset, formState } = useForm({
+    mode: "onChange",
     resolver: yupResolver(validationSchema),
   });
 
+  // routing
+  const history = useHistory();
+
   // log values when data is submitted
   const onSubmit = (values) => {
     console.log(values);
     reset();
+    history.push("/registration-complete");
   };
 
   // info for required entries
   const rEntries = [
     { name: "name", label: "Name" },
     { name: "email", label: "Email" },
-    { name: "number", label: "Phone Number" },
```

Comment:
Why isn't phone required anymore?

Reply:
The array was used to render the fields in a single section, but I had to separate the phone and notes parts into separate sections.